droidlet/lowlevel/locobot/remote/planning_service.py

```diff
@@ -76,27 +78,31 @@ def get_short_term_goal(self, robot_location, goal, step_size=25):
             target_goal = (stg_real[0], stg_real[1], rotation_angle)
         return target_goal
 
-    def goal_within_threshold(self, robot_location, goal):
-        distance = np.linalg.norm(np.array(robot_location[:2]) - np.array(goal[:2]))
-        # in metres. map_resolution is the resolution of the SLAM's 2D map, so the planner can't
-        # plan anything lower than this
-        threshold = (float(self.map_resolution) - 1e-10) / 100.0
+    def goal_within_threshold(self, robot_location, goal, threshold=None, angle_threshold=None):
```

Comment:
description would be good - although this one seems pretty self explanatory

Reply:
goal and location are the same type?

Reply:
goal and location are of same type, indeed.